display_module.rb

```diff
@@ -1,25 +1,63 @@
 module DisplayModule
   def list_all_books
-    @books.each { |book| puts "Title: #{book.title}, Author: #{book.author}" }
+    stored_books = load_books_from_json
+
+    if stored_books.empty?
+      puts 'No books found in the file.'
+    else
+      stored_books.each_with_index do |book, index|
+        puts "#{index}) Title: \"#{book['title']}\", Author: #{book['author']}"
+      end
+    end
+  end
+
+  def load_books_from_json
+    stored_books = []
+
+    if File.exist?('books.json')
+      existing_books_json = File.read('books.json')
+      stored_books = JSON.parse(existing_books_json) unless existing_books_json.empty?
+    end
+
+    stored_books
   end
 
   def list_all_people
-    @people.each { |person| puts "#{person.class}: Name: #{person.name}, Id: #{person.id}, Age: #{person.age}" }
+    if File.exist?('people.json')
+      people_json = File.read('people.json')
+      if people_json.empty?
+        puts 'The list is empty'
+      else
+        people_hash = JSON.parse(people_json)
+        @people = people_hash.map do |person|
+          if person['json_class'] == 'Student'
+            Student.from_hash(person, @classroom)
+          else
+            Teacher.from_hash(person)
+          end
+        end
+      end
+    end
```

Comment:
- [ ] It would be nice if you follow the project requirements because all the data should be saved when you quit the app :thinking: because right now your app saves the data into the files right after creating a person/book/rental which is not according to the requirements :fearful: Kindly make your app save the data into files after quitting the app :+1: